src/OrganisationRegistry.Api/Backoffice/Organisation/Requests/UpdateOrganisationRegulationRequest.cs

```diff
@@ -52,6 +53,17 @@ public UpdateOrganisationRegulationInternalRequestValidator()
             RuleFor(x => x.OrganisationId)
                 .NotEmpty()
                 .WithMessage("Organisation Id is required.");
+
+            RuleFor(x => x.Body.WorkRulesUrl)
+                .Custom(
+                    (workRulesUrl, context) =>
+                    {
+                        if (string.IsNullOrWhiteSpace(workRulesUrl))
+                            return;
+
+                        if (!Uri.TryCreate(workRulesUrl, UriKind.Absolute, out _))
+                            context.AddFailure("Arbeidsreglement moet een geldige url zijn.");
```

Comment:
This is good validation, but I'd like it to be in the domain (as well). Either in the handler or in the AR itself.